ui/tests/integration/components/raft-storage-overview-test.js

```diff
@@ -2,17 +2,56 @@ import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
 import { render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
 
 module('Integration | Component | raft-storage-overview', function (hooks) {
   setupRenderingTest(hooks);
 
-  test('it renders', async function (assert) {
-    let model = [
+  hooks.beforeEach(function () {
+    this.model = [
       { address: '127.0.0.1:8200', voter: true },
       { address: '127.0.0.1:8200', voter: true, leader: true },
     ];
-    this.set('model', model);
+  });
+
+  test('it renders', async function (assert) {
     await render(hbs`<RaftStorageOverview @model={{this.model}} />`);
     assert.dom('[data-raft-row]').exists({ count: 2 });
   });
+
+  test('it should download snapshot via service worker', async function (assert) {
```

Comment:
I couldn't come up with a nice way to test this end to end with the service worker in an appreciable amount of time so I added a test for the component to ensure that it's doing its part.